examples/rust/get_started/examples/06-secure-channel-many-hops.rs

```diff
@@ -16,15 +16,15 @@ async fn main(mut ctx: Context) -> Result<()> {
     ctx.start_worker("h3", Hop).await?;
 
     // Create a secure channel listener at address "secure_channel_listener"
-    bob.create_secure_channel_listener("secure_channel_listener")?;
+    bob.create_secure_channel_listener("secure_channel_listener", NoOpTrustPolicy)?;
```

Comment:
@jared-s Should we use IdentifierTrustPolicy here?

Reply:
Probably so, I'll add a note on Miro that we will want to switch to that. Nop is fine for this PR, we're going to redo all of the examples this week anway.